pkgs/development/compilers/ghc/head.nix

```diff
@@ -1,13 +1,13 @@
 { stdenv, fetchgit, bootPkgs, perl, gmp, ncurses, libiconv, binutils, coreutils
-, autoconf, automake, happy, alex, crossSystem, selfPkgs, cross ? null
+, autoconf, automake, happy, alex, python3, crossSystem, selfPkgs, cross ? null
```

Comment:
Why is python3 required? It might be a good idea to explain that change in the commit message.